cmake/RunSimulation.cmake

```diff
@@ -165,7 +165,7 @@ else()
   set(VALGRIND_COMMAND "")
 endif()
 
-
+# message(${VALGRIND_COMMAND} "${ENERGYPLUS_EXE}" -w "${EPW_PATH}" -d "${OUTPUT_DIR_PATH}" ${ENERGYPLUS_FLAGS_LIST} "${IDF_PATH}")
```

Comment:
@Myoldmopar Is this change supposed to be in this PR?

Reply:
I left it in there as a quick diagnostic line.  It's commented out so it doesn't do anything right now.